addons/cartel/units/CfgUnits.hpp

```diff
@@ -0,0 +1,633 @@
+// militia
+class GVAR(militia_aksu) : GVAR(Soldier_Base) {
+    scope = 2;
+    displayName = "Militia (AKSU)";
+    uniformClass = "U_I_C_Soldier_Bandit_2_F";
+    uksf_equipment_randomUniforms[] = {
+        "U_I_L_Uniform_01_tshirt_skull_F", 1,
+        "CUP_I_B_PMC_Unit_11", 1,
+        "U_C_E_LooterJacket_01_F", 1,
+        "U_C_ArtTShirt_01_v4_F", 1,
+        "CUP_O_TKI_Khet_Jeans_04", 1
+
+    };
+    weapons[] = { "arifle_AKS_F","Put","Throw" };
+    respawnWeapons[] = { "arifle_AKS_F","Put","Throw" };
+    items[] = { "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage" };
+    respawnItems[] = { "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_packingBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage", "ACE_elasticBandage" };
+    magazines[] = { "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "CUP_HandGrenade_RGD5", "CUP_HandGrenade_RGD5" };
+    respawnMagazines[] = { "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "30Rnd_545x39_Mag_Green_F", "CUP_HandGrenade_RGD5", "CUP_HandGrenade_RGD5" };
```

Comment:
Have you validated all of the items fit in the uniform+vest for every unit?

RPT will show a log like:

`soldier[<classname>]:Some of magazines weren't stored in soldier Vest or Uniform?`

for each class that has too many items for space.